packages/read-emails/client/react/src/App.js

```diff
@@ -24,8 +24,19 @@ function App() {
           console.error('An error occurred parsing the response:', error);
         });
     }
+    if (params.has('userId')) {
+      setUserId(params.get('userId'));
+    }
   }, [nylas]);
 
+  useEffect(() => {
+    if (userId.length) {
+      window.history.replaceState({}, '', `/?userId=${userId}`);
+    } else {
+      window.history.replaceState({}, '', '/');
+    }
+  }, [userId]);
+
```

Comment:
Just curious why this has been added back in?

Reply:
In order to update the url with userID which is used to a=identify the connected account in the local db